src/mscorlib/src/System/String.Comparison.cs

```diff
@@ -792,6 +792,25 @@ public int CompareTo(String strB)
             return false;
         }
 
+        [Pure]
+        public Boolean EndsWith(Char value)
+        {
+            return EndsWith(new string(value, 1), StringComparison.CurrentCulture);
```

Comment:
Do we actually need a culture sensitive `EndsWith` `char` overload? I'd bet that the people who asked for this didn't consider this aspect.